proposals/0000-improved-ghc-development.rst

```diff
@@ -0,0 +1,216 @@
+   * A full msys setup
+   * gdb/gcc etc for debugging
+   * check out Haskell sources
+   * do a local install of php and arc
+   * configure arc
+   * setup Hadrian
+   * setup SSH (for those who want to interface with it this way)
+ 
+  The script is set up to provide you with a working environment with little interaction needed. Setup is just one call. 
+
+  For Windows, such a script is a much better solution than Docker, simply because Docker does not work very well for Windows targets.
+
+- Phab reviewers list
+
+  Reviewers are currently assigned based on a set of Phabricator rules. As a maintainer on a less popular platform I know the pain of getting people to review your code.
+  
+  So my suggestion is to also assign reviewers based on the git history of the files you're changing. 
+
+  The reason for this is that currently it's always the same people reviewing patches.
+  Their time is spread thin. Particularly on less popular platforms it basically comes down to 4 people.
+  
+- Update trac linters
+
+  The git linters and Arc linters aren't currently checking the same things.
+
+  Particularly is the fact that the pre-commit hooks check
+  the summary text but arc doesn't. So commits can get rejected at push time requiring more fork for maintainers.
+
+  Also I want to say I love the summary document you have to fill in.
+  It ensures useful information is there later when I have to find out why a change was made. So whatever we do, don't remove this.
+
+- Phab signup screen
+
+  It's recently been made a requirement to require a public key to push to phab. The error you get when you don't do this and try to push a patch is very very cryptic and unintuitive. Could we make a plugin that asks the user to upload a public key on trac if they haven't done so? Like a banner at the top?
+
+- Add some automation to Trac (Or whatever bug tracker we end up using)
+
+  - Particular on new tickets post a friendly reminder that if they want they can give it a hand in fixing it themselves.
+  - Parse information added, in particular check if reproduction steps are there etc.
+  - If stack is used, kindly ask if a repo without can be used. The amount of bug reports with stack is increasing and regardless of my own opinion on the tool, these reports are not very useful as is.
+  - Maybe automatically CC people from a pool based on the information in the ticket? I tend to miss tickets because my filters are quite strict. Generally if the ticket doesn't mention my name, is directed at me or has "Windows" in the body somewhere it will skip my inbox. I review filtered tickets only once a week.
+  - If a newcomer assigns a ticket to themselves, have trac automatically post links to useful pages:
+  
+      - how to setup build environment.
+      - how to get help.
+      - assign a mentor?
+      - after x amount of time with no progress, remind them again that help is available
+
+- Unify the issue tracker and code review into phabricator
```

Comment:
I'm not sure to which part you are referring here. The issue tracker and code review unification; this would be the alternative to keeping trac for issues, and use maniphest instead.